cli/util/api.go

```diff
@@ -32,14 +32,14 @@ const (
 // of the API server (only used by the tests), in the order of precedence they
 // should be applied for the requested kind of node.
 func flagsForAPI(t repo.RepoType) []string {
```

Comment:
Could we change those repoType->thing functions into methods on RepoType? This way all of those per-type things are in one place.